pkg/controller/pxc/controller.go

```diff
@@ -254,14 +261,36 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 	return rr, nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) operatorPod(cr *api.PerconaXtraDBCluster) (corev1.Pod, error) {
+	operatorPod := corev1.Pod{}
+	err := r.client.Get(context.TODO(),
+		types.NamespacedName{
+			Namespace: cr.Namespace,
```

Comment:
operator and cr can be in different namespaces.
see https://github.com/operator-framework/operator-sdk/blob/master/pkg/k8sutil/k8sutil.go#L63-L77

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/363/commits/47a5705f0e5aa858f27b0506b6317a4a9e392c03